sanic/config.py

```diff
@@ -1,8 +1,19 @@
-import os
-import types
+from os import environ as os_environ
+
+from typing import Union, \
+                   Any
 
+from .utils import str_to_bool, \
+                   load_module_from_file_location
+
+# \/ \/ \/ \/
+# TODO: remove in version: 21.3
+import types
 from sanic.exceptions import PyFileError
 from sanic.helpers import import_string
+from warnings import warn
+# /\ /\ /\ /\ 
```

Comment:
Can it be like that ?

Reply:
Why do we need these characters `# /\ /\ /\ /\` ?

Reply:
> Make the comment more explicit as to what needs to be removed. Ideally someone who has never seen this PR would know what to do in January.

Reply:
Got it, sorry!

In that case, I suggest writing something like this example.

```
# NOTE(your_nickname): We should remove this function in the future release.
def deprecated_function():
    pass
```

Reply:
Please see new aproach.